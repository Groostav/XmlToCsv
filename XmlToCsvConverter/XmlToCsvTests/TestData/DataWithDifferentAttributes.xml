<?xml version="1.0" standalone="yes"?>
<DocumentElement>
  <DataWithDifferentAttributes 
    attribute="attribute1" attribute2="attribute1.1" 
	firstElement="true" singletonAttr="singleton for first element only">
    <ZIPCode attribute="Zip_attribute1" attribute2="Zip_attribute1.1">10001</ZIPCode>
    <CityName>James Town</CityName>
    <StateAbbr>NY</StateAbbr>
    <Country>USA</Country>
  </DataWithDifferentAttributes>
  <DataWithDifferentAttributes attribute="attribute2" attribute2="attribute1.2" secondElement="true">
    <ZIPCode attribute="Zip_attribute2" attribute2="Zip_attribute1.2">10002</ZIPCode>
    <CityName>Abbrevich</CityName>
    <StateAbbr>CA</StateAbbr>
    <Country>USA</Country>
  </DataWithDifferentAttributes>
  <DataWithDifferentAttributes attribute="attribute3" attribute2="attribute1.3">
    <ZIPCode attribute="Zip_attribute3" attribute2="Zip_attribute1.3">10003</ZIPCode>
    <CityName>Sommerville</CityName>
    <StateAbbr>WY</StateAbbr>
    <Country>USA</Country>
  </DataWithDifferentAttributes>
  <DataWithDifferentAttributes attribute="attribute4" attribute2="attribute1.4">
    <ZIPCode attribute="Zip_attribute4" attribute2="Zip_attribute1.4">10004</ZIPCode>
    <CityName>Loveland</CityName>
    <StateAbbr>TX</StateAbbr>
    <Country>USA</Country>
  </DataWithDifferentAttributes>
</DocumentElement>