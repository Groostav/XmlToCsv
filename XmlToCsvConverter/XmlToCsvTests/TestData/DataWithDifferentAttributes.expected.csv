DataWithDifferentAttributes_Id,CityName,StateAbbr,Country,attribute,attribute2,firstElement,singletonAttr,secondElement
0,"James Town","NY","USA","attribute1","attribute1.1","true","singleton for first element only",""
1,"Abbrevich","CA","USA","attribute2","attribute1.2","","","true"
2,"Sommerville","WY","USA","attribute3","attribute1.3","","",""
3,"Loveland","TX","USA","attribute4","attribute1.4","","",""
